openmetadata-service/src/main/java/org/openmetadata/service/migration/MigrationUtil.java

```diff
@@ -0,0 +1,360 @@
+package org.openmetadata.service.migration;
```

Comment:
shall we put them in appropriate versions? it would be better to name the versions/v110/mysql versions/v110/postgres v110/util